packages/components/src/components/iot-bar-chart/iot-bar-chart.tsx

```diff
@@ -34,6 +35,10 @@ export class IotBarChart {
 
   @Prop() styleSettings: StyleSettingsMap | undefined;
 
+  @Prop() audioAlertsEnabled = false;
```

Comment:
personal pref: "enableAudioAlerts"

Reply:
"enableAudioAlerts" follows the same pattern as `enableXScroll` in Synchro Charts, I'll change this :+1: